verifier/verifier.go

```diff
@@ -0,0 +1,114 @@
+package verifier
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/lestrrat-go/jwx/v2/jws"
+	"github.com/openpubkey/openpubkey/pktoken"
+)
+
+type Verifier struct {
+	providers map[string]*ProviderVerifier
+	cosigners map[string]*CosignerVerifier
+}
+
+func New(verifier *ProviderVerifier, options ...VerifierOpts) *Verifier {
```

Comment:
If only one verifier can be specified why is providers a list? How do you specify more than one provider?

Reply:
Using the Verifier options accepted in the "New" function